src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -68,12 +68,12 @@ void ExecutorWorker::execute() {
 
     // Keep a copy of the sorted list
     _opList = _syncPal->_syncOps->opSortedList();
-
+    auto perfMonitor = SentryHandler::ScopedPTrace(SentryHandler::PTraceName::InitProgress, syncDbId());
```

Comment:
Not sure to understand how it works...
If we want the `initProgress`method to be evaluated, shouldn't we call the ScopedTrace inside to method.

And therefor calling ScopedTrace at higher level here like `PTraceName::Propagation2`?